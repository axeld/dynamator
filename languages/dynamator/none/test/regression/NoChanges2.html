<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<html>
<head>
  <title>Test No Changes</title>
</head>
<body>
<p>Actually, this tests for the browser whitespace bug
<p>
&#187; &#188;
&#8192;&nbsp;&#8192;

<table>
  <tr>
    <td><img src="x"></td>
    <td><img src="x"> 
    </td>
  </tr>
</table>
<a href="xxx">text</a>
<br>
<a href="xxx">text
</a><!-- tidy fixes this last one -->
<br>
The end.
</body>
</html>
