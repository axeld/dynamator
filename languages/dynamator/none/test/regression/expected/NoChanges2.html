<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<html>
  <head>
    <title>Test No Changes</title>
  </head>
  <body>
    <p>Actually, this tests for the browser whitespace bug</p>
    <p>&#187; &#188; &#8192;&nbsp;&#8192;</p>
    <table>
      <tr>
        <td><img src="x"></td>
        <td><img src="x"> </td>
      </tr>
    </table>
    <a href="xxx">text</a> <br>
    <a href="xxx">text</a> <!-- tidy fixes this last one -->
    <br>
    The end.
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
